pages/wiki/openembedded.md

```diff
@@ -3,10 +3,243 @@ title: OpenEmbedded
+meta-python           /home/developer/asteroid/src/meta-openembedded/meta-python  5
+meta-filesystems      /home/developer/asteroid/src/meta-openembedded/meta-filesystems  5
+meta-anthias          /home/developer/asteroid/src/meta-smartwatch/meta-anthias  7
+meta-bass             /home/developer/asteroid/src/meta-smartwatch/meta-bass  7
+meta-beluga           /home/developer/asteroid/src/meta-smartwatch/meta-beluga  7
+meta-catfish          /home/developer/asteroid/src/meta-smartwatch/meta-catfish  7
+meta-dory             /home/developer/asteroid/src/meta-smartwatch/meta-dory  7
+meta-lenok            /home/developer/asteroid/src/meta-smartwatch/meta-lenok  8
+meta-mooneye          /home/developer/asteroid/src/meta-smartwatch/meta-mooneye  7
+meta-mtk6580          /home/developer/asteroid/src/meta-smartwatch/meta-mtk6580  7
+meta-narwhal          /home/developer/asteroid/src/meta-smartwatch/meta-narwhal  7
+meta-ray              /home/developer/asteroid/src/meta-smartwatch/meta-ray  7
+meta-sawfish          /home/developer/asteroid/src/meta-smartwatch/meta-sawfish  7
+meta-skipjack         /home/developer/asteroid/src/meta-smartwatch/meta-skipjack  6
+meta-smelt            /home/developer/asteroid/src/meta-smartwatch/meta-smelt  7
+meta-sparrow          /home/developer/asteroid/src/meta-smartwatch/meta-sparrow  7
+meta-sprat            /home/developer/asteroid/src/meta-smartwatch/meta-sprat  7
+meta-sturgeon         /home/developer/asteroid/src/meta-smartwatch/meta-sturgeon  7
+meta-swift            /home/developer/asteroid/src/meta-smartwatch/meta-swift  8
+meta-tetra            /home/developer/asteroid/src/meta-smartwatch/meta-tetra  7
+meta-wren             /home/developer/asteroid/src/meta-smartwatch/meta-wren  7
+```
+
+By convention, Yocto layers all begin with `meta`. There are layers for individual pieces such as Qt, networking, Python, and filesystems. These are followed by the layers for each of the watch codenames. If you are looking for an AsteroidOS-specific application, you can probably guess that it's in the `meta-asteroid` layer, but you can also ask the computer to search for you.
+
+Originally, `bitbake` and its related tools were the only way to do things, but there is now a newer and much more convenient tool named [`devtool`](https://docs.yoctoproject.org/dev/ref-manual/devtool-reference.html). To search for a recipe with `flashlight` in the name:
+
+```
+devtool search flashlight
+```
+
+After a minute or so, `devtool` finds a single recipe: `asteroid-flashlight`. You can now modify this recipe:
+
+
+```
+devtool modify asteroid-flashlight
+```
+
+The last two lines of output will be something like these:
+
+```
+INFO: Source tree extracted to /home/developer/asteroid/build/workspace/sources/asteroid-flashlight
+INFO: Recipe asteroid-flashlight now set up to build from /home/developer/asteroid/build/workspace/sources/asteroid-flashlight
+
+```
```

Comment:
For the formatting to be correct, shouldn't the three backticks be one line higher ? (and then the empty line after the three backticks)

Reply:
It's not strictly necessary, but it does make the output a bit neater, so I've made that change.